test/regression/generateRegressionTests.ts

```diff
@@ -1,59 +1,61 @@
 import * as child_process from "child_process";
 
-import {$, snap, Snappit, IConfig} from "snappit-visual-regression";
-import {WebDriver, By, WebElement, WebElementPromise} from "selenium-webdriver";
+import {$, By, snap, Snappit, IConfig, WebDriver, WebElement} from "snappit-visual-regression";
 
 import {test, TestContext} from "ava";
 
 import * as util from "../common/util";
 
-let snappit: Snappit;
-let driver: WebDriver;
-
-const grepCommand = 'grep "\\bdata-visreg=" ../source/components/**/*.html';
+const grepCommand = 'grep "\\bdata-visreg=" ../docs/**/**/*.html';
```

Comment:
Will this not work with a single set of double asterisks? `../docs/**/*.html` vs `../docs/**/**/*.html`?

Reply:
I can use

```
grep "\\bdata-visreg=" -r ../docs/
```

But as far as I know, I can't scope it to just html files without forcing it to glob through sub directories manually.

Reply:
```
grep "\\bdata-visreg=" -r ../docs | grep "index.html" 
```

would do it though.